packages/pn-commons/package.json

```diff
@@ -71,16 +75,12 @@
     "eslint-plugin-react": "^7.27.1",
     "eslint-plugin-react-hooks": "^4.3.0",
     "eslint-plugin-sonarjs": "^0.10.0",
-    "jest-sonar-reporter": "^2.0.0",
-    "npm-run-all": "^4.1.5",
+    "identity-obj-proxy": "^3.0.0",
```

Comment:
I guess it can be removed. I added it to avoid problems running the test *under Jest*. Create React App injects a Jest configuration that smoothes the execution of tests; when I removed CRA, I obtained a number of test failures, and the way I found to solve this is to replicate some parts of the configuration injected by CRA. 
There is a reference in https://pagopa.atlassian.net/wiki/spaces/PN/pages/823459883/Migrazione+di+Create+React+App+verso+Vite.
I attach the configuration that CRA injects into test execution, I obtained it by running Jest with an option that outputs the config it uses.
[pn-7438-config-react-script.txt](https://github.com/pagopa/pn-frontend/files/13874699/pn-7438-config-react-script.txt)